config/src/main/java/org/hyperledger/besu/config/QbftConfigOptions.java

```diff
@@ -23,21 +23,22 @@
 public class QbftConfigOptions extends BftConfigOptions {
   public static final QbftConfigOptions DEFAULT =
       new QbftConfigOptions(JsonUtil.createEmptyObjectNode());
+  public static final String VALIDATOR_CONTRACT_ADDRESS = "validatorcontractaddress";
```

Comment:
Could this and `QbftFork.VALIDATOR_CONTRACT_ADDRESS_KEY` be combined or are they semantically different?

Reply:
👍🏼

Reply:
`QbftFork.VALIDATOR_CONTRACT_ADDRESS_KEY` can be used here instead?

Reply:
It could be used here, not sure I like QbftConfigOptions depending on constants from the QbftFork though

Reply:
Think if anything it should be the other way around. QbftFork is really a subset of the QbftConfigOptions. Rather push this change into another PR and then we could fix all the keys at once. This PR is already getting rather large.

Reply:
👍🏼